applications/GeoMechanicsApplication/custom_utilities/functions_for_parameters.cpp

```diff
@@ -0,0 +1,62 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Anne van de Graaf,
+//                   Marjan Fathian,
+//                   Gennady Markelov
+//
+
+#include "custom_utilities/functions_for_parameters.h" 
+
+namespace Kratos {
+
+    Parameters FunctonsForParameters::ExtractParameters(const Parameters& rSourceParameters,
+                                                        const std::vector<std::string>& rNamesOfParametersToCopy)
+    {
+        auto result = Parameters{};
+        result.CopyValuesFromExistingParameters(rSourceParameters, rNamesOfParametersToCopy);
+        return result;
+    }
+
+    void FunctonsForParameters::AppendParameterNameIfExists(const std::string& rParameterName,
```

Comment:
Minor comment about the styling: The functions in this file don't need to be indented here, inside namespaces we don't use indentation

Reply:
Thank you. It is removed now.